tests/sls/common/timezone.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+
+from tests.util import Failure, Success
+
+
+def run():
+    command = "date | grep -v UTC"
+    ret = subprocess.run(command,
+                         stdout=subprocess.PIPE,
+                         stderr=subprocess.PIPE,
+                         universal_newlines=True,
+                         shell=True)
```

Comment:
Please remove `shell=True`, it can cause security problems (and also spawns an extra process).